iron-form.html

```diff
@@ -277,6 +277,12 @@
           return;
         }
 
+        // Ensure the native form fired the `reset` event.
```

Comment:
can you add a bit more details about who could've fired the reset event? or what other event it can be?

Reply:
Done